### ISSUE TITLE
Add alt text to labels in app explore page

### ISSUE THREAD

**Issue by Noah-Huppert** [kscout/kscout.io#80]
A user should be able to hover over a tag on a card in the app explore page and see some alt text which says "filter by <tag>".

Action items:

- [ ] Add alt text to tags in app cards on app explore page

Acceptance criteria:

- [ ] (1) Hovering over tags in app cards on the app explore page should show alt text indicating that clicking on the tag will filter by the tag
